### PR TITLE
commit ex16

### PR DESCRIPTION
**Pull request by MichaelMayes7**

### CODE REVIEW

File: src/main/java/com/xpanxion/java/springboot/da1/demo/model/student6/Gym.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.xpanxion.java.springboot.da1.demo.model.student6;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="GYM")
+public class Gym {
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/springboot/da1/demo/model/student6/Gym.java:9]: Can you please change this from `Gym` to `Gym6`?

Comment (MichaelMayes7) [reply src/main/java/com/xpanxion/java/springboot/da1/demo/model/student6/Gym.java:9]: sure thing should i do the same with service, repo etc?

File: src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student6/GymController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GymController {
+
+    @Autowired
+    GymService gymService;
+
+    @GetMapping("student6/api/v1/gym{id}")
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student6/GymController.java:15]: The exercise states that we need to create this endpoint: `student{id}/api/v1/gym?name={gym name here}`

File: src/main/java/com/xpanxion/java/springboot/da1/demo/model/student6/Gym.java (hunk 1 above)
Comment (mburolla) [reply src/main/java/com/xpanxion/java/springboot/da1/demo/model/student6/Gym.java:9]: Yes, please add your student id to anything that has a SpringBoot annotation.

**Pull request merged**